pkg/types/types.go

```diff
@@ -99,6 +99,10 @@ type DataSource struct {
 	ID   SourceID `json:",omitempty"`
 	Name string   `json:",omitempty"`
 	URL  string   `json:",omitempty"`
+
+	// BaseID shows Base source of advisories.
+	// e.g. `Root.io` based on Debian/Ubuntu/Alpine advisories.
+	BaseID SourceID `json:",omitempty"`
```

Comment:
`omitzero` is equivalent to `omitempty` in case of string, but there's no reason to use `omitempty` after [Go 1.24](https://github.com/aquasecurity/trivy-db/pull/556).

```suggestion
	BaseID SourceID `json:",omitzero"`
```